src/components/shared/Tooltip.js

```diff
@@ -26,44 +24,38 @@ type State = {
 
 export default class Tooltip extends React.PureComponent<Props, State> {
   _isMounted: boolean = false;
-  _mountElement: ?HTMLElement;
+  _mountElement: HTMLElement;
+  _tooltipRoot: ?HTMLElement;
```

Comment:
I think we only need the tooltipRoot, see how greg did it for the screenshots: https://github.com/devtools-html/perf.html/blob/d61cc1207accd923833ca20783be085449a5d0cf/src/components/timeline/TrackScreenshots.js#L62-L65

This would make it possible to remove some more code as we don't need to add or remove the element because React does it for us.

Reply:
I didn't read the code, but my general preference on portals is to have a single overlay element, that we then can control the z-indexing explicitly between the various components.

Reply:
Actually the solution to https://github.com/devtools-html/perf.html/issues/1274 is especially to have several root overlay elements :) Because we want tooltips to go over all of the chrome, but not screenshots because they get in the way.

Reply:
I suppose I can just use `#root-overlay` for this issue since it seems like the tooltips and screenshots won't display at the same time, and thus there's no conflict between them. Today I learned about stacking contexts! :tada:

Reply:
There's no conflict between screenshots and tooltips, _but_ we want them to have a different behavior relatively to stacking context. I think we _need_ different root points; although we can look at it later when dealing with #1274.